diff -up hypervkvpd-0/hv_kvp_daemon.c.daemon hypervkvpd-0/hv_kvp_daemon.c
--- hypervkvpd-0/hv_kvp_daemon.c.daemon	2013-08-09 10:51:41.020041192 +0200
+++ hypervkvpd-0/hv_kvp_daemon.c	2013-08-09 10:53:23.217056215 +0200
@@ -1434,8 +1434,6 @@ int main(void)
 	char *kvp_recv_buffer;
 	size_t kvp_recv_buffer_len;
 
-	if (daemon(1, 0))
-		return 1;
 	openlog("KVP", 0, LOG_USER);
 	syslog(LOG_INFO, "KVP starting; pid is:%d", getpid());
 
