diff -up ./hv_vss_daemon.c.daemon ./hv_vss_daemon.c
--- ./hv_vss_daemon.c.daemon	2013-06-26 10:26:42.272419971 +0200
+++ ./hv_vss_daemon.c	2013-06-26 10:27:12.894443174 +0200
@@ -148,9 +148,6 @@ int main(void)
 	char *vss_recv_buffer;
 	size_t vss_recv_buffer_len;
 
-	if (daemon(1, 0))
-		return 1;
-
 	openlog("Hyper-V VSS", 0, LOG_USER);
 	syslog(LOG_INFO, "VSS starting; pid is:%d", getpid());
 
